portal-web/docroot/html/taglib/ui/input_localized/page.jsp

```diff
@@ -116,9 +116,9 @@
 				languageValue = ParamUtil.getString(request, name + StringPool.UNDERLINE + curLanguageId, languageValue);
 			}
 
-			if (curLanguageId.equals(defaultLanguageId) && Validator.isNull(languageValue)) {
+			/*if (curLanguageId.equals(defaultLanguageId) && Validator.isNull(languageValue)) {
 				languageValue = LocalizationUtil.getLocalization(xml, defaultLanguageId, true);
-			}
+			}*/
 		%>
 
```

Comment:
@crodriguezy, this looks weird. Since the change is in the tag, except with the current ticket, does other usages exist the issue? It looks unrelated with the current issue. If the issue still exists in other usages, please create one new ticket to fix it.

Reply:
I've separeted it in a new bug, please see https://github.com/liferay-core-infra/liferay-portal/pull/2398